src/compas_blender/artists/meshartist.py

```diff
@@ -255,13 +262,15 @@ def draw(self):
         if self.show_edges:
             self.draw_edges()
 
-    def draw_mesh(self):
+    def draw_mesh(self) -> List[bpy.types.Object]:
         """Draw the mesh."""
         vertices, faces = self.mesh.to_vertices_and_faces()
         obj = compas_blender.draw_mesh(vertices, faces, name=self.mesh.name, collection=self.collection)
         return [obj]
 
-    def draw_vertices(self, vertices=None, color=None):
+    def draw_vertices(self,
+                      vertices: Optional[List[int]] = None,
+                      color: Optional[str, Tuple, List, Dict] = None) -> List[bpy.types.Object]:
```

Comment:
i guess this could be made more specific by defining a color type as a tuple of 3 `float` or 3 `int` and then use those inside `List` and `Dict`

Reply:
something like
```python
Color = Union[Tuple[int, int, int], Tuple[float, float, float]]
...
color: Optional[str, Color, List[Color], Dict[int, Color]]
```